src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -699,6 +699,11 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'default': defaultWindowTitleSeparator,
 				'markdownDescription': localize("window.titleSeparator", "Separator used by {0}.", '`#window.title#`')
 			},
+			'window.titleAccessibilityOptimized': {
```

Comment:
Don't we typically put accessibility options under the umbrealla of `accessibility.*`?